core/io/resource.h

```diff
@@ -74,6 +74,9 @@ class Resource : public RefCounted {
 
 	SelfList<Resource> remapped_list;
 
+	void _dupe_sub_resources(Variant &r_variant, Node *p_for_scene, HashMap<Ref<Resource>, Ref<Resource>> &r_remap_cache);
+	void _find_sub_resources(const Variant &r_variant, HashSet<Ref<Resource>> &r_resources_found);
```

Comment:
`r_` stands for parameters passed as pointer/reference that get modified inside the function (like the variant you are replacing). For other parameters you should use `p_` prefix.

Reply:
So it should be `p_variant` in the second one, as it's not mutable

Reply:
Also `p_hashmap`. The contents are changed, the HashMap itself is the same.